test/unit/shadow/polyfill.js

```diff
@@ -81,6 +81,13 @@ describe('shadow/polyfill', function () {
     it('innerHTML', function () {
       root.innerHTML = '<slot name="test"></slot>';
       expect(root.childNodes[0].name).to.equal('test');
+
+      if (canPatchNativeAccessors) {
```

Comment:
Happy for this to go in like this, but it would seem we're going to need to do this non-Safari stuff while still testing in Safari more and more. Ideas on how to do this are welcome. I'm not sure this would be maintainable in the long run.